features/20-exchange-access-token.feature

```diff
@@ -5,13 +5,20 @@ Feature: App exchanges authorization code for access token
         Given OAuth is enabled
         And I am not logged in
         When I ask for authorization
-        And I exchange my authorization code
+        And I exchange my authorization code via a POST request
```

Comment:
I might just default this  value to POST since it is the only correct thing to do and it will avoid polluting the various places in which we issue this as an ordinary step along the way to some unrelated test (rather than a specific negative test)